tests/integration/integration_test.go

```diff
@@ -236,254 +235,6 @@ func TestRemoveWithResourcesBlocked(t *testing.T) {
 	e.RunCommand("pulumi", "stack", "rm", "--yes")
 }
 
-// TestPreviewJSON tests serializing a preview as JSON using the `pulumi preview --json` option.
```

Comment:
Just out of curiosity, what diffs were you seeing here? I know you mentioned spacing, but I'm surprised that it wasn't possible to get that to match--I've updated this test before without similar issues.

Reply:
No matter what formatting I gave these tests, it was incorrect! Spaces / tabs - I even grabbed everything via JSON Lint and added it and it was still wrong :(